migrations/v2.js

```diff
@@ -0,0 +1,176 @@
+import { describe, getCourse, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('Page level progress - v2.0.1 to v2.0.2', async () => {
+  // https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/compare/v2.0.1..v2.0.2
+
+  let course, coursePlpGlobals;
+  const plpOptionalContentNew = 'Optional Content';
+  const plpIndicatorBarDefaultOld = 'You have completed ';
+  const plpIndicatorBarDefaultNew = 'Progress bar. Select here to view your current progress, and select an item to navigate to it. You have completed ';
+
+  whereFromPlugin('Page level progress - from v2.0.1', { name: 'adapt-contrib-pageLevelProgress', version: '>=2.0.0 <2.0.2' });
+
+  whereContent('Page level progress is configured', content => {
+    course = getCourse();
+    return course._pageLevelProgress;
+  });
+
+  mutateContent('Page level progress - add globals if missing', async (content) => {
+    if (!_.has(course, '_globals._extensions._pageLevelProgress')) _.set(course, '_globals._extensions._pageLevelProgress', {});
```

Comment:
Is this the same issue with not adding earlier missing globals defaults for a subsequent update?

![image](https://github.com/user-attachments/assets/f87a8d81-eece-47ff-b424-feae1a579a6e)
![image](https://github.com/user-attachments/assets/c4a5ef91-3fac-476c-8f45-d9abd779eddf)
![image](https://github.com/user-attachments/assets/1ec61387-cc94-4974-b987-e32eb3e0e4f7)

Should it be? 
![image](https://github.com/user-attachments/assets/b3698a97-52d9-4b2b-8b14-78f58d7a6329)
![image](https://github.com/user-attachments/assets/c4a5ef91-3fac-476c-8f45-d9abd779eddf)
![image](https://github.com/user-attachments/assets/5c82d2d1-c2f9-421c-80fe-c402d68be52a)


So that the finishing state is as expected, with all three new globals added if they weren't there before?

```suggestion
    if (!_.has(course, '_globals._extensions._pageLevelProgress')) {
      _.set(course, '_globals._extensions._pageLevelProgress', {
        pageLevelProgressIndicatorBar: plpIndicatorBarDefaultOld
      });
    }
```